src/clr-angular/layout/tabs/_tabs.clarity.scss

```diff
@@ -27,3 +27,48 @@ button.nav-link {
     margin-right: 0;
   }
 }
+
+.tabs-vertical {
```

Comment:
So the big hangup here is that if you do static tabs with vertical, they require an extra container. We can mask that with Angular but not in static HTML. Is my understanding correct?

Reply:
Yes, Jeremy, absolutely correct. The static HTML has not a single root, but is handled as a collection of elements, all on root level. Although it's not needed for the component functioning, I think it is not a bad idea if we aim for all our components to have a single root.
The website and dev/ks-apps are updated with appropriate examples, which you can see later in this review.